src/index.js

```diff
@@ -1 +1,46 @@
-export default "TODO";
+import * as babylon from "babylon";
+
+// Parse options values.
+const _parseOpts = (opts) => Object.keys(opts)
+  // Parse the matched replacement expression and
+  // https://github.com/babel/babylon/issues/210
+  .map((key) =>
+    [key, babylon.parse(`expr=${opts[key]}`).program.body.pop().expression.right])
```

Comment:
After reading through the linked issue, there doesn't seem to exist a fantastic alternative until/unless the PR is merged.

![](https://media.giphy.com/media/ALalVMOVR8Qw/giphy.gif)